### PR TITLE
changed vmalert behaviour

### PR DESCRIPTION
**Pull request by f41gh7**
added ability to start with empty rules directory

### CODE REVIEW

**Comment by valyala** [comment]
@f41gh7 , could you extend commit message with the explanation why this change may be needed?

@hagen1778 , PTAL

File: app/vmalert/config/config.go [hunk 1]
```diff
@@ -2,6 +2,7 @@ package config
 
 import (
 	"fmt"
+	"github.com/VictoriaMetrics/VictoriaMetrics/lib/logger"
```
[the remark this answers is not in the document]
Comment (f41gh7) [reply app/vmalert/config/config.go]: done

File: app/vmalert/config/config.go [hunk 2]
```diff
@@ -9,6 +9,8 @@ import (
 	"strings"
 	"time"
 
+	"github.com/VictoriaMetrics/VictoriaMetrics/lib/logger"
+
```
Comment (hagen1778) [inline app/vmalert/config/config.go:13]: extra new line may be omitted

File: app/vmalert/manager_test.go [hunk 3]
```diff
@@ -23,12 +22,8 @@ func TestManagerUpdateError(t *testing.T) {
 	m := &manager{groups: make(map[uint64]*Group)}
```
Comment (hagen1778) [inline app/vmalert/manager_test.go:22]: Please rename test to "TestManagerEmptyGroups" or similar to reflect the purpose of this test.

File: app/vmalert/manager_test.go [hunk 4]
```diff
@@ -23,12 +22,8 @@ func TestManagerUpdateError(t *testing.T) {
...
 	path := []string{"foo/bar"}
 	err := m.update(context.Background(), path, true, true, false)
-	if err == nil {
-		t.Fatalf("expected to have err; got nil instead")
-	}
-	expErr := "no groups found"
-	if !strings.Contains(err.Error(), expErr) {
-		t.Fatalf("expected to got err %s; got %s", expErr, err)
+	if err != nil {
+		t.Fatalf("expected to load succesfully with empty rules dir; got err instead: %v", err)
```
Comment (hagen1778) [inline app/vmalert/manager_test.go:26]: Please add a new test or a test case to `TestManagerUpdate` to verify that `vmalert` can be started with empty groups and then updated with existing ones.

**Review by hagen1778 — CHANGES_REQUESTED** [review]
LGTM! Pls note my comments

**Review by hagen1778 — APPROVED** [review]

**Review by valyala — APPROVED** [review]
